ui/Screen/Org.svelte

```diff
@@ -113,7 +121,8 @@
         <ProjectsMenu
           orgAddress={address}
           {gnosisSafeAddress}
-          disabled={activeTab.projectCount === 0} />
+          disabled={activeTab.projectCount === 0 || pendingAnchor}
```

Comment:
Now that the logic to disable this button is a bit more complex we should just pass in project count to the `ProjectsMenu` component as a prop and do all the logic inside the component.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2085/commits/8f523c9e166bcdf4b4f99eb94b50bb0580ec17c7